lib/block_view.go

```diff
@@ -1607,6 +1607,10 @@ func _isEntryImmatureBlockReward(utxoEntry *UtxoEntry, blockHeight uint32, param
 	return false
 }
 
+func (bav *UtxoView) VerifySignature(txn *MsgDeSoTxn, blockHeight uint32) (_derivedPkBytes []byte, _err error) {
```

Comment:
We need it to be public if we intend to separate UtxoView and PosMempool into their own modules -- which I think we should do eventually. I chose to make the wrapper because renaming `_verifySignature` to `VerifySignature` will result in a lot of changes to existing code, unnecessarily.